lib/dao.js

```diff
@@ -1602,10 +1669,17 @@ DataAccessObject.count = function (where, options, cb) {
     options: options
   };
   this.notifyObserversOf('access', context, function(err, ctx) {
-      if (err) return cb(err);
-      where = ctx.query.where;
-      Model.getDataSource().connector.count(Model.modelName, cb, where);
-    });
+    if (err) return cb(err);
+    where = ctx.query.where;
+
+    if (connector.count.length <= 3) {
+      // Old signature, please note where is the last
```

Comment:
Would you mind rewording the comment to make it more clear that the word "where" is referring to an argument name, and it is not a part of the sentence? E.g.

```js
// Old signature, please note "where" is the last argument
```

I had to read the current comment twice to understand the meaning. Although this may be just me.